substrate/frame/revive/fixtures/Cargo.toml

```diff
@@ -1,6 +1,5 @@
 [package]
 name = "pallet-revive-fixtures"
-publish = true
```

Comment:
You will need to update the umbrella crate as well to get ci to pass

Reply:
Why? `publish = true` is a noop and this is why I removed it.

Reply:
ok nvm then, I thought I thought the umbrella python script  was filtering crate based on the publish flag